common/constants/shared.ts

```diff
@@ -84,6 +88,10 @@ export const observabilityDataConnectionsPluginOrder = 9030;
 export const queryWorkbenchPluginID = 'opensearch-query-workbench';
 export const queryWorkbenchPluginCheck = 'plugin:queryWorkbenchDashboards';
 
+export const alertsID = 'alerting';
+export const anomalyDetectionID = 'anomaly-detection-dashboards';
+export const discoverID = 'data-explorer/discover';
```

Comment:
can make variable names more descriptive. `alertsID` -> `alertingPluginId`

Reply:
<img width="544" alt="Screenshot 2024-09-03 at 1 31 51 PM" src="https://github.com/user-attachments/assets/077c4044-e26d-46b0-9d94-8765f85a8589">
Added 'Plugin' to all variables.